### ISSUE TITLE
Previous sprint Retrospective boards are not available and newly created Retrospective is not visible to any of the team member 

### ISSUE THREAD

**Issue by muthu3758** [microsoft/vsts-extension-retrospectives#842]
**Describe the bug**

1. Historical Retro boards are not visible on the board rather it's available only in the History tab
2. Newly created Retro board is not visible to any of the team mebers.

**Are you using the Retrospectives Tool on-prem?**
[ ] No

If Yes, what version of the extension?

I'm using in AzureDevOps Services

**Comment by PratMoha** [comment]
Same issue, the previous ticket was closed without resolving the issue.

**Comment by JFriedman09** [comment]
> Same issue, the previous ticket was closed without resolving the issue.

Yes. I checked our boards today for retrospectives and the historical boards that were available a couple of weeks ago are still only in history.

**Comment by lukasz969** [comment]
we face the same bug in 2 projects in 2 organizations
AzureDevOps Services
Installed ext version 1.92.17 (Latest)
Last updated Sep 13, 2024 at 12:16 AM GMT+2

**Comment by muthu3758** [comment]
![image](https://github.com/user-attachments/assets/4382c26f-d75d-40dc-96c9-1d75fbe5713f)

I see that latest updates in the extension happened on Sep 13, 2024 at 12:16 AM GMT+2, this could be the reason for this issue.

Kindly check it.

**Comment by dphanson63** [comment]
The latest release includes a new option under Settings to "Show all teams" which should resolve this issue.  I suspect what was happening is that some users were not a member of the team, so they might see all boards under History, but only boards where they were a team member, under Boards.
